museum/sampleRecords/donors.destination.json

```diff
@@ -0,0 +1,578 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "entity_id":"1",
+      "id":"1",
+      "idno":"1",
+      "display_label":"N\/A",
+      "ca_entities.idno":"1",
+      "ca_entities.preferred_labels.displayname":"N\/A",
+      "ca_entities.preferred_labels.surname":"&lt;&gt;"
```

Comment:
This appears to be an artefact of the web services: http://staging-collections.histwest.org.au/index.php/editor/entities/EntityEditor/Edit/entity_id/1